README.md

```diff
@@ -123,6 +123,14 @@ To run tests, run the following command:
 
 - Project is deployed using [GitHub Pages](https://github.com/techmoves/todo-list.git)
```

Comment:
- [ ] As mentioned in the previous review, the link redirecting users to GitHub pages is misleading, instead it leads to the GitHub repo